src/components/home/elements/IssueToken.vue

```diff
@@ -0,0 +1,758 @@
+        <b-container v-show="coldPageId===4">
+          <TokenConfirm :address="address"
+                        :amount=inputAmount(coldAmount)
+                        :fee="coldFee"
+                        :tx-type="'Issue Token'">
+          </TokenConfirm>
+          <p v-show="sendError">Sorry, transaction send failed!</p>
+          <b-row>
+            <b-col class="col-lef">
+              <b-button
+                class="btn-back"
+                block
+                variant="light"
+                size="lg"
+                @click="coldPrevPage">Back
+              </b-button>
+            </b-col>
+            <b-col class="col-rit">
+              <b-button
+                block
+                class="btn-confirm"
+                variant="warning"
+                size="lg"
+                @click="sendData('coldWallet')">Confirm
+              </b-button>
+            </b-col>
+          </b-row>
+        </b-container>
+        <b-container v-show="coldPageId===5">
+          <TokenSuccess class="tokenSucced"
+                        :address="address"
+                        :amount=inputAmount(coldAmount)
+                        :fee="coldFee"
+                        :tx-type="'Issue Token'">
+          </TokenSuccess>
+          <b-button variant="warning"
+                    block
+                    size="lg"
+                    @click="endSend">OK
+          </b-button>
+        </b-container>
+      </b-tab>
+    </b-tabs>
+  </b-modal>
+</template>
+
+<script>
+import Vue from 'vue'
+import seedLib from '@/libs/seed.js'
+import { NODE_IP, CONTRACT_EXEC_FEE, ISSUE_FUNCIDX, TRANSFER_ATTACHMENT_BYTE_LIMIT, VSYS_PRECISION, FEE_SCALE, API_VERSION, PROTOCOL, OPC_ACCOUNT, OPC_FUNCTION } from '@/constants.js'
+import TokenConfirm from '../modals/TokenConfirm'
+import TokenSuccess from '../modals/TokenSuccess'
+import ColdSignature from '../modals/ColdSignature'
+import browser from '../../../utils/browser'
+import BigNumber from 'bignumber.js'
+import transaction from '@/utils/transaction'
+export default {
+    name: 'IssueToken',
+    components: {ColdSignature, TokenSuccess, TokenConfirm},
+    data: function() {
+        return {
+            amount: BigNumber(0),
+            coldAmount: BigNumber(0),
+            attachment: '',
+            pageId: 1,
+            fee: BigNumber(CONTRACT_EXEC_FEE),
+            coldPageId: 5,
+            coldFee: BigNumber(CONTRACT_EXEC_FEE),
+            // address: this ? (this.walletType === 'hotWallet' ? this.selectedAddress : this.defaultAddress) : '',
+            coldAddress: this ? (this.walletType === 'coldWallet' ? this.selectedAddress : this.defaultColdAddress) : '',
+            scanShow: false,
+            sendError: false,
+            coldSignature: '',
+            timeStamp: Date.now() * 1e6,
+            hasConfirmed: false
+        }
+    },
+    props: {
+        tokenBalance: {
+            type: BigNumber,
+            default: function() {
```

Comment:
why no default value here but in BurnToken.vue line 296 should use `BigNumber(0)`?

Reply:
fixed in commit. Add BigNumber(0)